index.d.ts

```diff
@@ -58,18 +66,19 @@ declare namespace Moleculer {
 		static createFromPayload(broker: ServiceBroker, payload: object): Context;
 	}
 
-	interface ServiceActionSchema {
-		cache?: boolean;
-		params?: object;
-		handler: (ctx: Context) => Promise<any>;
-	}
-
 	interface ServiceSettingSchema {
 		port?: number | string;
-		routes?: Route[];
+		routes?: RouteSchema[];
+		transport?: string | Transporter;
```

Comment:
`transport` is not a service setting as well.

Reply:
Then, service setting has only: 
```
[name: string]: any;
```
is it true?

Reply:
Yes, but there are two internal optional settings: https://moleculer.services/docs/service.html#Internal-settings